libraries/AP_EFI/AP_EFI_Scripting.cpp

```diff
@@ -0,0 +1,18 @@
+#include "AP_EFI_Scripting.h"
+
+#if AP_EFI_SCRIPTING_ENABLED
+
+// Called from frontend to update with the readings received by handler
+void AP_EFI_Scripting::update()
+{
+    // Nothing to do here
```

Comment:
Should probably have some sort of health check that times out if we have not heard from the script in a while.

Reply:
is_healthy() already handles that as it has an update time

Reply:
I guess we have to trust the script writer to populate it correctly in that case.